### PR TITLE
example updates

### PR DESCRIPTION
**Pull request by friedenberg**
two parts to this PR:

- example comment updates in one commit
- addition of a `finalUrl` parameter for the AddPerformanceMaxRetailCampaiign example

Apologies that they were merged into this one, I messed up the branches I was working on. I would normally do two separate PR's for this.

### CODE REVIEW

**Review by sundquist — APPROVED** [review]
LGTM

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 1]
```diff
@@ -497,7 +497,7 @@ public MutateOperation CreateRoot()
 
             string assetGroupResourceName = ResourceNames.AssetGroup(customerId, assetGroupId);
 
-            // We use a factory create all the MutateOperation's that manipulate a specific
+            // We use a factory to create all the MutateOperation's that manipulate a specific
```
Comment (jradcliff) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:500]: nit: Do we need the apostrophe on `MutateOperation's`?

**Review by jradcliff — APPROVED** [review]

**Review by AnashOommen — APPROVED** [review]

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 2]
```diff
@@ -119,8 +119,8 @@ public static void Main(string[] args)
 
                     // The option to remove the existing listing group tree.
                     //
-                    // If the current AssetGroup already has a tree of ListingGroupFilter's, and you
-                    // try to add a new set of ListinGroupFilter's including a root filter, you'll
+                    // If the current AssetGroup already has a tree of ListingGroupFilters, and you
+                    // try to add a new set of ListinGroupFilters including a root filter, you'll
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:123]: Typo in "ListinGroupFilters"

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 3]
```diff
@@ -119,8 +119,8 @@ public static void Main(string[] args)
...
                     // receive a 'ASSET_GROUP_LISTING_GROUP_FILTER_ERROR_MULTIPLE_ROOTS' error.
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:124]: should be receive *an* 'ASSET...

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 4]
```diff
@@ -477,6 +476,7 @@ public MutateOperation CreateRoot()
             // [END add_performance_max_product_listing_group_tree_6]
         }
 
+        // [START add_performance_max_product_listing_group_tree]
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:479]: I think you're missing the end tag for this.

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 5]
```diff
@@ -497,13 +497,13 @@ public MutateOperation CreateRoot()
...
-            // are used to optionally remove all AssetGroupListingGroupFilter's from the tree, and
+            // are used to optionally remove all AssetGroupListingGroupFilters from the tree, and
             // then to construct a new tree of filters. These filters can have a parent-child
             // relationship, and also include a special root that includes all children.
             //
-            // When creating these filters, we use temporary ID's to create the hierarchy between
+            // When creating these filters, we use temporary IDs to create the hierarchy between
             // the root listing group filter, and the subdivisions and leave nodes beneath that.
             //
             // The factory specific to a customerId and assetGroupId is created below.
```
[the remark this answers is not in the document]
Comment (devchas) [reply examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:509]: We may also want to consider renaming this variable to something like "otherSubdivisionId"

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 6]
```diff
@@ -226,12 +225,12 @@ public List<MutateOperation> RemoveAll()
 
             // [START add_performance_max_product_listing_group_tree_3]
             /// <summary>
-            /// Creates a list of MutateOperation's that remove all the descendents of the specified
+            /// Creates a list of MutateOperations that remove all the descendents of the specified
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:228]: descendents is spelled incorrectly -> descendants

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 7]
```diff
@@ -214,10 +213,10 @@ private class AssetGroupListingGroupFilterRemoveOperationFactory {
...
             /// <summary>
-            /// Creates a list of MutateOperation's that remove all of the resources in the tree
+            /// Creates a list of MutateOperations that remove all of the resources in the tree
             /// originally used to create this factory object.
             /// </summary>
-            /// <returns>A list of MutateOperation's</returns>
+            /// <returns>A list of MutateOperations</returns>
             public List<MutateOperation> RemoveAll()
             {
                 return this.RemoveDescendentsAndFilter(this.rootResourceName);
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:222]: descendants also spelled incorrectly here (and in the call to this method)

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 8]
```diff
@@ -152,10 +151,10 @@ public static void Main(string[] args)
...
-        /// A factory that creates MutateOperation's for removing an existing tree of
-        /// AssetGroupListingGroupFilter's.
+        /// A factory that creates MutateOperations for removing an existing tree of
+        /// AssetGroupListingGroupFilters.
         ///
-        /// AssetGroupListingGroupFilter's must be removed in a specific order: all of the children
+        /// AssetGroupListingGroupFilters must be removed in a specific order: all of the children
         /// of a filter must be removed before the filter itself, otherwise the API will return an
         /// error.
         ///
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:160]: Is the [resources](https://github.com/googleads/google-ads-dotnet/blob/6f0678260aa7913865f1c0aab24bd61ccaef1d06/examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs#L166) instance variable actually used? Can this be removed?

Comment (BenRKarl) [reply examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:160]: +1 it doesn't appear to be used anywhere.

**Pull request merged**